lib/keystore.js

```diff
@@ -57,17 +60,22 @@ function nacl_decodeHex(msgHex) {
 }
 
 
-var KeyStore = function(mnemonic, pwDerivedKey, hdPathString) {
-
-  this.defaultHdPathString = "m/0'/0'/0'";
+var KeyStore = function(mnemonic, pwDerivedKey, hdPathString, salt) {
```

Comment:
We could add a `console.warn` about the deprecation, too, to help spread the word.

> On Sep 3, 2016, at 9:19 AM, Christian Lundkvist <notifications@github.com> wrote:
> 
> In lib/keystore.js <https://github.com/ConsenSys/eth-lightwallet/pull/113#discussion_r77438907>:
> 
> > @@ -57,17 +60,22 @@ function nacl_decodeHex(msgHex) {
> >  }
> >  
> >  
> > -var KeyStore = function(mnemonic, pwDerivedKey, hdPathString) {
> > -
> > -  this.defaultHdPathString = "m/0'/0'/0'";
> > +var KeyStore = function(mnemonic, pwDerivedKey, hdPathString, salt) {
> I think we should probably leave the old constructor as it is, but add a warning that it will soon be deprecated and please use the new constructor.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/ConsenSys/eth-lightwallet/pull/113/files/0c8f7be3281f39762aaee2bcc4c44e59bfc62be1#r77438907>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAhIjyg6yx4x9IsUjO4w6R7er78sfA8Mks5qmZ4EgaJpZM4JyK7d>.
>